src/frontend/src/pages/myfeeds.js

```diff
@@ -208,7 +215,9 @@ export default function MyFeeds() {
                           color={submitStatus.isError ? 'error' : 'textPrimary'}
                           align="center"
                         >
-                          {submitStatus.message}
+                          {alert === true && submitStatus.message !== '' ? (
+                            <CustomizedSnackBar message={submitStatus.message} open={alert} />
+                          ) : null}
```

Comment:
Can this use of `CustomizedSnackBar` be moved outside of (or simply replace) the Typography tag that currently encloses it? I'm receiving a lot of these warnings in the console:

![image](https://user-images.githubusercontent.com/13500769/79088762-89eabd80-7d11-11ea-85ab-09877ca7890d.png)